selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/factory/DirectUrlFactoryWithSpecificBranchTest.java

```diff
@@ -52,25 +55,35 @@
   @Inject private TestWorkspaceServiceClient workspaceServiceClient;
   @Inject private TestProjectServiceClient testProjectServiceClient;
   @Inject private PullRequestPanel pullRequestPanel;
+  @Inject private TestGitHubRepository testRepo;
 
   private TestFactory testFactoryWithSpecificBranch;
 
   @BeforeClass
   public void setUp() throws Exception {
+    // preconditions - create test the repository
```

Comment:
Test repo has actually created earlier, and here we are adding project into the test repo.